api/src/main/java/uk/gov/dvsa/mot/motr/model/DvlaVehicleWithLatestTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.dvsa.mot.motr.model;
+
+import uk.gov.dvsa.mot.motr.service.DateConverter;
+import uk.gov.dvsa.mot.trade.api.DvlaVehicle;
+
+import java.time.LocalDate;
+import java.time.Year;
+
+import javax.validation.constraints.NotNull;
+
+import static java.util.Objects.requireNonNull;
+
+public class DvlaVehicleWithLatestTest implements VehicleWithLatestTest {
```

Comment:
A bit confusing name format for those objects, since in this instance there are no tests. Maybe something like `VehicleWithLatestTestExpiry` or `VehicleWithLatestTestData`. Or rename only `DvlaVehicleWithLatestTest` to reflect reality

Reply:
Changed to DvlaVehicle (though the name is alerady used).